library/modules/bigip_wait.py

```diff
@@ -23,6 +23,13 @@
     of rebooting due to a configuration change.
 version_added: 2.5
 options:
+  type:
+    description:
+      - The type of the BIGIP defaults to standard, the other option being VCMP.
```

Comment:
mention the choices and what each choice is for, something like this:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_cgnat_lsn_pool.py#L48

Reply:
Updated docs.